.circleci/Dockerfile.8.15

```diff
@@ -1,14 +1,16 @@
-FROM coqorg/coq:8.15.2-ocaml-4.14.0-flambda
+FROM coqorg/coq:8.15
```

Comment:
`-ocaml-` tags are unstable. Use the default tag unless its OCaml version doesn't satisfy QuickChick.